### ISSUE TITLE
only calculate label placements for landuse layer at later zooms

### ISSUE THREAD

**Issue by nvkelso** [tilezen/vector-datasource#679]
**tl;dr** Olga already moved most important landuse features to the pois layer. What remains there should only receive calculated label placements at later zooms. I propose zoom 15+.

===

Looks like we need ensure all landuse kinds in the lookup table are represented **both** in the [landuse.csv](https://github.com/mapzen/vector-datasource/blob/master/spreadsheets/kind/landuse.csv) table and in the [pois.csv](https://github.com/mapzen/vector-datasource/blob/master/spreadsheets/kind/pois.csv) table. Right now the several `park` lookups are only in the landuse table (not the pois table), so attraction wins when it does get to the `pois` layer, or they are orphaned in the `landuse` layer if there is no equivalent rule in the `pois` layer. 

The proposed rule of thumb is: The landuse, buildings, and water layers should generally only have label placements points for things that don't get icons (are shown with text only labels). 

* **What did you see?** (1) In tile [16/19299/24630](http://vector.mapzen.com/osm/all/16/19299/24630.topojson?api_key=vector-tiles-HqUVidw) (download [zip](
https://github.com/mapzen/vector-datasource/files/204439/24630.json.zip)) I expected to see this park label feature (`id: 218327131`) to show up `pois` layer. (2) Similarly I expected to see this park label feature (`id:22727025`) to show up as `kind:park` in the `pois` layer.
* **What did you expect to see?** Instead I see (1) `218327131`'s label position in the `landuse` layer, and (2) `22727025` as a `kind:attraction` in the `pois` layer.
* **What map location are you having problems with?** New York City, near http://tangrams.github.io/bubble-wrap/#18/40.75397/-73.98267. 
* **Screenshot?** 

<img width="965" alt="screen shot 2016-04-05 at 08 33 16" src="https://cloud.githubusercontent.com/assets/853051/14281794/1a6ba6f6-fb09-11e5-892c-9cffd17fec0a.png">

**1) for park in landuse > pois:**

```
{
type: "Polygon",
properties: {
kind: "park",
name: "Special Permit Plaza",
area: 2045,
sort_key: 24,
source: "openstreetmap.org",
min_zoom: 11.757,
id: 218327131
},
arcs: [
[
502
]
]
},
{
type: "Point",
properties: {
label_placement: "yes",
kind: "park",
name: "Special Permit Plaza",
area: 2045,
sort_key: 24,
source: "openstreetmap.org",
min_zoom: 11.757,
id: 218327131
},
coordinates: [
417,
815
]
```

2) park or attraction

```
{
type: "Point",
properties: {
id: 22727025,
kind: "attraction",
min_zoom: 17,
name: "Bryant Park",
area: 40577
},
coordinates: [
720,
574
]
},
```

**Comment by nvkelso** [comment]
I don't expect to see landuse labels that are only label_placements befoe zoom 16:

- http://tile.dev.mapzen.com/vector/v1/all/16/10480/25323.topojson?api_key=vector-tiles-HqUVidw

```
type: "Point",
properties: {
label_placement: true,
kind: "pier",
name: "Municipal Pier",
area: 13097,
sort_key: 228,
min_zoom: 10.4176,
source: "openstreetmap.org",
id: 82206919
},
coordinates: [
930,
949
]
```

**Comment by nvkelso** [comment]
I suspect:

```
  - fn: vectordatasource.transform.generate_label_features
    params:
      source_layer: landuse
      start_zoom: 15
      label_property_name: label_placement
      label_property_value: true
```

**Comment by nvkelso** [comment]
WFM
